package.json

```diff
@@ -22,26 +22,24 @@
   "homepage": "https://github.com/anthonygore/html-critical-webpack-plugin#readme",
   "scripts": {
     "clean": "rimraf ./dist && rimraf ./test/cases/**/build",
-    "build": "npm run clean && npm run lint && webpack --config ./webpack.config.js",
```

Comment:
could you share why this script was removed?  Just cautious of having a lot of changes here when upgrading a key dependency if it's not actually related to upgrading the dependency.